evaluation.md

```diff
@@ -2,8 +2,8 @@
 
 ### Back-end
 
-- Your API includes at least 2 models, one of which may be a user
-- Your models include at least one association
+- Your API includes at least 2 models, backed by MongoDB (one of which may be a user)
+- Your models include at some form of 'association', e.g. nested documents
```

Comment:
feel like user auth should be required either via 3rd part api or local auth - we're teaching 3 classes on it